helm/alert-logic/templates/daemonset.yaml

```diff
@@ -20,7 +20,8 @@ spec:
         visualize: "true"
     spec:
       tolerations:
-      - operator: "Exists"
+      - key: node-role.kubernetes.io/master
```

Comment:
I think that the "Exists" operator should be kept in order to deploy it on all the nodes.